controllers/feeds/getFeedsV2.js

```diff
@@ -151,7 +151,7 @@ module.exports = async (req, res) => {
 
         const feeds = await getActivtiesOnFeed(feed, token, paramGetFeeds);
         if (feeds.length === 0) {
-          if (feed === 'main_feed_broad') {
+          if (feed === 'main_feed') {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the condition from `'main_feed_broad'` to `'main_feed'` might cause a different behavior if the `feed` variable never equals `'main_feed'`. If the loop was supposed to terminate when `feed` equals `'main_feed_broad'`, it will now continue indefinitely until `feed` equals `'main_feed'`. Please ensure that this change aligns with your intended logic.
```diff
-          if (feed === 'main_feed_broad') {
+          if (feed === 'main_feed') {
             break;
           } else {
             offset = 0;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->